ml-agents-protobuf/make.bat

```diff
@@ -0,0 +1,43 @@
+# variables
+
+# GRPC-TOOLS required. Install with `nuget install Grpc.Tools`. 
+# Then un-comment and replace [DIRECTORY] with location of files.
+# For example, on macOS, you might have something like:
+COMPILER=Grpc.Tools.1.14.1/tools/macosx_x64
```

Comment:
Maybe let the default be `[Directory]` ?

Reply:
Woops, meant to comment that line out. I've gone ahead and done it.